test/aws-sqs/yaks-config.yaml

```diff
@@ -1,9 +1,13 @@
 config:
+  namespace:
+    temporary: true
   runtime:
     resources:
-      - ../../aws-sqs-source.kamelet.yaml
       - aws-sqs-client.yaml
 pre:
+  - name: Install kamel
+    run: |
+      kamel install -w -n ${YAKS_NAMESPACE} --maven-repository=http://nexus.fuse-qe.eng.rdu2.redhat.com/repository/fuse-all --olm=false
```

Comment:
@lfabriko please don't use our internal nexus. We can add it in our pipeline.

Reply:
I will remove it also from other yaks config files.

Reply:
Thanks. Remove the `--olm=false` flag as well, please.